client/app/bundles/course/user-invitations/operations.ts

```diff
@@ -129,3 +130,43 @@ export function toggleRegistrationCode(shouldEnable: boolean): Operation {
         );
       });
 }
+
+export const splitEntries = (input: string): string[] => {
+  return input.split(/\s*[;,]\s*(?=(?:[^"]*"[^"]*")*[^"]*$)/);
+};
```

Comment:
this is because the purpose of `splitEntries` and `parseEmails` are actually very different, even though the returning value is a bit similar

- `splitEntries` is given a string, you split it into chunks of entries that contains single information regarding name and email
- `parseEmails`, on the other hand, took `splitEntries` inside its function indeed, but its main purpose is to, given an entry of name and email (in an agreed format), to parse it so that we can get the properly formatted entry of {name, entry}

I will refactor this so that the naming is proper